lib/ui/classical_components/barcode_custom_ui.dart

```diff
@@ -249,11 +250,11 @@ class _BarcodeScannerWidgetState extends State<BarcodeScannerWidget> {
                 );
               }),
           showProgressBar
-              ? Center(
-                  child: Container(
+              ? const Center(
+                  child: SizedBox(
                     width: 100,
                     height: 100,
-                    child: const CircularProgressIndicator(
+                    child: CircularProgressIndicator(
```

Comment:
And, this can also be const, too I guess

Reply:
This is done due to the automatic flutter format . and dart fix